core/server/api/v2/utils/validators/input/schemas/posts.json

```diff
@@ -133,67 +133,77 @@
             "description": "Authors of the post",
             "type": "array",
             "items": {
-                "type": "object",
-                "properties": {
-                    "id": {
-                        "type": "string",
-                        "maxLength": 24
+                "anyOf": [{
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "string",
+                            "maxLength": 24
+                        },
+                        "slug": {
+                            "type": "string",
+                            "maxLength": 191
+                        },
+                        "email": {
+                            "type": "string",
+                            "maxLength": 191
+                        },
+                        "roles": {
+                            "strip": true
+                        },
+                        "permissions": {
+                            "strip": true
+                        }
                     },
-                    "slug": {
-                        "type": "string",
-                        "maxLength": 191
-                    },
-                    "email": {
-                        "type": "string",
-                        "maxLength": 191
-                    },
-                    "roles": {
-                        "strip": true
-                    },
-                    "permissions": {
-                        "strip": true
-                    }
-                },
-                "anyOf": [
-                    {"required": ["id"]},
-                    {"required": ["slug"]},
-                    {"required": ["email"]}
-                ]
+                    "anyOf": [
+                        {"required": ["id"]},
+                        {"required": ["slug"]},
+                        {"required": ["email"]}
+                    ]
+                }, {
+                    "type": "string",
+                    "maxLength": 191
+                }]
             }
         },
         "post-tags": {
             "description": "Tags of the post",
             "type": "array",
             "items": {
-                "type": "object",
-                "properties": {
-                    "id": {
-                        "type": "string",
-                        "maxLength": 24
-                    },
-                    "name": {
-                        "type": "string",
-                        "maxLength": 191
+                "anyOf": [{
```

Comment:
Alternative is: `allOf`, but I think it's more flexible if you can mix the values.

Reply:
No that does not work